examples/async/async.js

```diff
@@ -166,29 +166,17 @@ document.addEventListener('click', (e) => {
   if(e.target.matches('.button-click')) {
 
     aa('click', {
+      eventName: "hit-clicked",
+      indexName: process.env.INDEX_NAME,
       objectID: e.target.getAttribute('data-objectid'),
       position: parseInt(e.target.getAttribute('data-position'))
     })
 
   } else if(e.target.matches('.button-convert')) {
 
     aa('conversion', {
-      objectID: e.target.getAttribute('data-objectid')
-    })
-  }
-})
-
-document.addEventListener('contextmenu', (e) => {
```

Comment:
I simplified the example and removed the context-menu listener here.

Reply:
That's cool, do you even think it's worth having a context-menu example? I think it makes sense, especially as some people I imagine search for something and just open everything in a new tab. In that case we might be missing data and it could be quite significant. I would advocate to have this in our doc tho, wdyt?

Reply:
> I think it makes sense, especially as some people I imagine search for something and just open everything in a new tab

oh! you're right. Now I understand the point of having that listener.
We should definitely add it to our docs!

I wonder if there's anything more precise: receiving a context menu event might not mean the user open it in a new tab. I guess it was hard to find a better solution?

Reply:
ok, i've opened an issue so that we don't forget it: https://github.com/algolia/search-insights.js/issues/26